GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -642,6 +644,11 @@ private void OnScroll()
             _lastScroll.Restart();
         }
 
+        private void TriggerRowCountUpdate()
```

Comment:
Why do we need this added level of indirection?

Reply:
> Why do we need this added level of indirection?

>> which will be really split in a follow up

In this step, I want to clarify *why* we update - namely for really different reasons. In follow-ups, only necessary things will be done here. In a final step, the graph will be rendered to the cache bitmap in a background task. Then the expensive `BuildOrderedRowCache` and `BuildSegmentLanes` will not delay / not need to be executed by the UI thread.